src/hooks/auth/useLogout.ts

```diff
@@ -0,0 +1,13 @@
+import { useThirdwebConfigContext } from "../../contexts/thirdweb-config";
+import invariant from "tiny-invariant";
+
+export function useLogout() {
+  const { authUrl } = useThirdwebConfigContext();
+
+  function logout() {
+    invariant(authUrl, "Please specify an authUrl in the ThirdwebProvider");
+    window.location.href = `${authUrl}/logout`;
+  }
+
+  return logout;
```

Comment:
slight improvement here, you can wrap this in a `useCallback` to make it not re-create the function on every render (unless the `authUrl` changes)